include/Oscillator.h

```diff
@@ -163,7 +163,7 @@ class LMMS_EXPORT Oscillator
 
 	static inline sample_t noiseSample( const float )
 	{
-		return 1.0f - rand() * 2.0f / RAND_MAX;
+		return 1.0f - rand() * 2.0f / static_cast<float>(RAND_MAX);
```

Comment:
I'll split this into it's own PR. This fix is to suppress an implicit conversion warning.